Hi3Helper.Core/Lang/en.json

```diff
@@ -203,7 +203,7 @@
     "Graphics_Title": "Graphics Settings",
     "Graphics_ResolutionPanel": "Game Resolution",
     "Graphics_Fullscreen": "Fullscreen",
-    "Graphics_ExclusiveFullscreen": "Use Exclusive Fullscreen",
+    "Graphics_ExclusiveFullscreen": "Use Exclusive Fullscreen (Borderless Windowed)",
```

Comment:
I don't think Borderless Windowed is the same as Exclusive Fullscreen. As Exclusive Fullscreen will take over the entire display (noticeable by brief flicker when Alt-Tabbing).

Reply:
You're right, I'll make the necessary changes.